package.json

```diff
@@ -24,7 +24,9 @@
   "keywords": [],
   "author": "",
   "license": "ISC",
-  "dependencies": {},
+  "dependencies": {
+    "react-router-dom": "^5.1.2"
```

Comment:
I assume you remembered to install this *before* it broke your code, unlike me--good job!